src/components/WorkOrder/Photos/ControlledFileInput.tsx

```diff
@@ -41,26 +42,21 @@ const ControlledFileInput = (props: Props) => {
   } = props
 
   const inputRef = useRef<HTMLInputElement | null>(null)
-  const [isCompressing, setIsCompressing] = useState(false)
-  const [totalFilesToCompress, setTotalFilesToCompress] = useState(0)
-  const [processedCount, setProcessedCount] = useState(0)
   const [previewFiles, setPreviewFiles] = useState<File[]>([])
```

Comment:
There was too much interdependent state which was probably the cause of the UI bugs - by using the actual previewFiles count as the source of truth for the progress this ensures the UI is in sync with the real progress - easier code logic too